client/src/components/CatalogTourCard/CatalogTourCard.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { Stack, CardActions, CardMedia, Button, Typography, CardContent } from '@mui/material';
+import AccessTimeIcon from '@mui/icons-material/AccessTime';
+import { ReactComponent as CoinsIcon } from '../../svg/CoinsIcon.svg';
+import styles from './CatalogTourCard.module.scss';
+
+function CatalogTourCard() {
+  return (
+    <Stack
+      direction={{ xs: 'column', tablet: 'row' }}
+      sx={{ backgroundColor: 'white', borderRadius: '10px', boxShadow: '0px 0px 40px rgba(0, 0, 0, 0.05)' }}
+      spacing={1}
+    >
+      <CardMedia
+        sx={{
+          width: {
+            xs: '100%',
+            tablet: '238px',
+          },
+          height: {
+            xs: '240px',
+            tablet: 'auto',
+          },
+          borderRadius: {
+            xs: '10px 10px 0 0',
+            tablet: '10px 0 0 10px',
+          },
+        }}
+        component='img'
+        image='https://visitukraine.today/media/tours/gallery/ALeR7GgYjAqCqfJnQX5ZYKnBcsDZ6MTJs77IIBKi.jpg'
+        alt='tour photo'
+      />
+      <CardContent sx={{ padding: '0 30px 0 36px' }}>
+        <Typography
+          sx={{ color: 'black', marginBottom: '20px', marginTop: '20px' }}
+          variant='h3'
+          component='div'
+        >
+          Sightseeing tour of Chernivtsi
+        </Typography>
+
+        <Stack direction='row' spacing={3} alignItems='start' sx={{ marginBottom: '20px' }}>
+          <Stack direction='row' spacing={1} alignItems='center'>
+            <CoinsIcon />
+            <Typography variant='h3' color='primary'>
```

Comment:
here you go,  you are using color : primary which is mui theme syntax.  but on the other hand you could delete the color property here since h3 is already styled by the primary color by default in our theme

Reply:
done